Mathlib/AlgebraicTopology/SplitSimplicialObject.lean

```diff
@@ -258,11 +258,18 @@ attribute [instance] Splitting.map_isIso
 #align simplicial_object.splitting.map_is_iso SimplicialObject.Splitting.map_isIso
 
 /-- The isomorphism on simplices given by the axiom `Splitting.map_isIso` -/
-@[simps!]
 def iso (Δ : SimplexCategoryᵒᵖ) : coprod s.N Δ ≅ X.obj Δ :=
   asIso (Splitting.map X s.ι Δ)
 #align simplicial_object.splitting.iso SimplicialObject.Splitting.iso
 
+@[simp]
+theorem iso_hom (Δ : SimplexCategoryᵒᵖ) : (iso s Δ).hom = Splitting.map X s.ι Δ :=
+  rfl
+
+@[simp]
+theorem iso_inv (Δ : SimplexCategoryᵒᵖ) : (iso s Δ).inv = inv (Splitting.map X s.ι Δ) :=
+  rfl
+
```

Comment:
Why this change? This seems like it should work.

Reply:
Because `Splitting.map` is unfolded in `simps` lemmas.
I wonder why `Splitting.map` isn't unfolded in the old Lean.

Reply:
Okay, I added a comment to that effect.